serverenvironmentexample.adoc

```diff
@@ -0,0 +1,155 @@
+
+Initial Configuration on HTTP Server
+------------------------------------
+1. Replace `grr/install_data/etc/server.local.yaml` and
+`grr/install_data/etc/grr-server.yaml` with the files from main server
+
+Configuration on Master Dataserver
+----------------------------------
+1. Replace `grr/install_data/etc/server.local.yaml` and
+`grr/install_data/etc/grr-server.yaml` with the files from main server
+2. Edit `grr/install_data/etc/server.local.yaml` and add the following:
++
+[source,yaml]
+Datastore.location: <full path to datastore>
+Dataserver.server_list:
+  - http://<dataserver ip>:7000
+Dataserver.client_credentials:
+  - <client_username>:<client_password>:rw
+Dataserver.server_username: <server_username>
+Dataserver.server_password: <server_password>
+
+* Terms:
+    - `server_list` is a list of all database servers with the master listed
+      first
+    - `client_credentials` are used for the http server and main server to
+      communicate with the database
+    - `server_username` and `server_password` are used for any slave dataservers
+      to talk to the master dataserver.
+3. Start the dataserver service by running `grr_server --component
+dataserver_master`
+* It should start with no errors and not show any information. If you would
+ like to see more information add in `--verbose`
+
+Finish the Configuration on the Main Server
+-------------------------------------------
+1. Edit install_data/etc/server.local.yaml
+* Dataserver Configuration:
++
+[source,yaml]
+Dataserver.server_list:
+   - http://<dataserver ip>:7000
+Datastore.implementation: HTTPServer
+HTTPDatastore.username: <client_username>
+HTTPDatastore.password: <client_password>
+
+* Email Configuration:
++
+[source,yaml]
+Worker.smtp_server: <smtp server>
+Worker.smtp_port: <smtp port>
+Worker.smtp_starttls: True
+Worker.smtp_user: <username>
+Worker.smtp_password: <password>
+Email.approval_cc_address: <monitoring email>
+
+- *Note* `approval_cc_address` should only be the account name, your email
+ domain will be appended later.
+* Private Key configuration 
++
+[source,yaml]
+PrivateKeys.executable_signing_private_key: "%(<path to key>|file)"
+
+2. Add users who should be able to access the admin interface.
+* *Note* Users cannot change their password in the admin ui, they must change
+ it in the command line
+* https://github.com/google/grr-doc/blob/master/admin.adoc#user-management[User
+ Management]
+3. Start services with `grr_server --component worker and grr_server --component
+ui`
+
+Finish the configuration on HTTP Server
+---------------------------------------
+1. Edit `install_data/etc/server.local.yaml`
++
+[source,yaml]
+Dataserver Configuration:
+Dataserver.server_list:
+   - http://<dataserver ip>:7000
+Datastore.implementation: HTTPServer
+HTTPDatastore.username: <client_username>
+HTTPDatastore.password: <client_password>
+
+2. Start services with `grr_server --component http_server`
+
```

Comment:
The next few sections look like clarifications on setting up the HTTP datastore, based on people already knowing about the blogpost. I'd be more inclined to replace the entire blogpost with a new install page in the docs. If you are willing to take that on (since you just went through it all), we can put a deprecation notice on the blogpost and point people to the doc. What do you think?